lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -6,7 +6,13 @@ class Processor::MetricsFileInfo < Processor
       # @type self: SchemaClass
       let :issue, object(
         lines_of_code: integer?,
-        last_committed_at: string
+        last_committed_at: string,
+        number_of_churn_commits: integer,
+        churn: object?(
+          occurrence: integer,
+          additions: integer,
+          deletions: integer
+        )
```

Comment:
[question] Is there an advantage of this nested data structure? Does it make sense to use a flat one instead?

```suggestion
        occurrence: integer,
        additions: integer,
        deletions: integer,
```

I'm worried that the client program will be not simple if we use the nested structure.

Reply:
Fixed. commit e999765f64242dca0d9595d139f7e5398fdf404a